### ISSUE TITLE
Using a JPG on a an HDRi Skybox results in a black cubemap

### ISSUE THREAD

**Issue by Kadino** [o3de/o3de#2850]
**Describe the bug**

**To Reproduce**
1. Open the Entity Outliner
2. Expand 'Atom Default Environment'
3. Select 'Global Sky'
4. In the 'HDRi Sskybox' component, change the texture to a jpg
5. See error

**Expected behavior**
Component can accept simple non-EXR image types and make a best guess at how it projects onto a cube, or another skybox component exists for non-EXR images

**Screenshots**
![skybox](https://user-images.githubusercontent.com/830514/128252263-5188b526-9bab-4057-9008-d5187058e2a4.png)

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Win
 - Version 10
 - CPU Intel i7-9750H
 - GPU NVidia RTX 2070
 - Memory 32GB

**Additional context**

**Comment by Jackerty** [comment]
I had a similar issue with a DDS file. After discussion on the discord actual problem, at least for me, was the file's name. I did not know about [Name Conventions](https://www.o3de.org/docs/user-guide/assets/texture-settings/texture-presets/) regarding textures.

Still testing what happens to JPGs with the right naming convention.

**Comment by ParasiteSt** [comment]
@Kadino I was able to get JPG working for HDRi skybox, but like Jackerty was saying you need to add the naming convention to the end of your asset name. For example if your JPG HDRi skybox was named Skybox you would rename it to Skybox_iblglobalcm
and import that into o3de.

**Comment by Kadino** [comment]
There's a lot of options for making this more friendly, e.g. the user could be prompted to have the file automatically renamed and configured as a working skybox.

Recommend discussing the behavior and naming conventions with @o3de/sig-ui-ux

**Comment by ParasiteSt** [comment]
I agree there should be some automation to this process, but also if you are trying to use a JPG for a skybox I normally would not see any engine asking if it is a skybox as JPG's are more commonly known for textures. Maybe more of a ui that shows up on import that has dropdowns for assigning what the asset is suppose to be for, for example look at the fbx import ui widget that you get with Unreal Engine.
